### DISCUSSION TITLE
Twitter Advertising

### DISCUSSION THREAD

**Discussion by alephate** [evennia/evennia#2843] [General Discussion]
A few of us were thinking that it would be good to spread word about our developing projects on Twitter, and in doing so, more widely publicize MUDs as a whole as well. We can follow and shout out to each other and have a twitter support circle!

https://twitter.com/mystavaria

https://twitter.com/songofavaria

https://twitter.com/WorldOfGlain

If you have a twitter page for your MUD, feel free to post it here!

**Comment by aMiss-aWry** [comment]
https://twitter.com/MachineGarden
